stopcovid/drills/drills.py

```diff
@@ -15,7 +15,7 @@ class Prompt(pydantic.BaseModel):
     messages: List[PromptMessage]
     response_user_profile_key: Optional[str] = None
     correct_response: Optional[str] = None
-    max_failures: Optional[int] = 1
+    max_failures: int = 1
```

Comment:
I checked the`current_prompt` in the dialog state table, and verified we are defaulting to 1 when we send the prompts over from scadmin.